data/human/deep jobs.txt

```diff
@@ -353,3 +353,617 @@ mission "Record academic conference"
+			has "main plot completed"
+	on offer
+		log "Minor People" "Agent Wayde" `A young agent in the logistics division of Deep Security, he vouched for your increased job opportunities after you helped him out.`
+		conversation
+			`As you walk away from a food stand in the spaceport, a rather young man in activewear steps away from the wall and smoothly sidles up next to you, between one step and the next.`
+			`	"Hello Captain. Please don't be alarmed, do you have a moment?"`
+			`	His words come out very quickly as if they'd been heavily rehearsed. Glancing around, so far as you can tell, no one appears to be paying any attention to either of you.`
+			choice
+				`	"Sure. Who are you?"`
+					goto who
+				`	"Sorry, but I'm busy right now."`
+			`	"Uh, I was coming to thank you if that helps?" he says.`
+			choice
+				`	"Oh. What is it?"`
+				`	"Sorry, maybe next time."`
+					decline
+			
+			label who
+			`	He continues walking and speaks without looking directly at you. "My name is Wayde, Agent Wayde if you will. Don't mind the title though, I'm uh, well I work in the logistics division for Deep Security. You've transported a large number of secure packages recently and several of them were mine."`
+			choice
+				`	"Work is work."`
+					goto blindacceptance
+				`	"Ah, yes."`
+					goto blindacceptance
+				`	"Do all the secure packages belong to Deep Security?"`
+			`	"Oh no, plenty of tech businesses and wealthy individuals use the same secure package shipping channels. We do monitor all of them though just to make sure our own channels don't get compromised by something ill-advised... or illegal." He shrugs.`
+			label blindacceptance
+			`	"Well, I looked at your record and it seems you've done quite a lot for the Deep, and you're well-regarded. We like to keep a list of pilots that we can call on if we need something delivered with either urgency or discretion, and it seems like you qualified. You passed the background check at any rate.`
+			`	"So at any rate, I'm authorized to give you a one-time offer to join that list. Don't worry, I'm not asking anything of you right now."`
+			`	Belatedly he seems to do a double-check that no one is actively listening to the conversation as you walk. "So, you interested?"`
+			choice
+				`	"Oh sure, I'm willing to help."`
+					goto agree
+				`	"Sorry, but I don't think I want to get that involved."`
+					decline
+				`	"You said 'at any rate' twice."`
+			`	He looks away sheepishly. "Er, yes, sorry about that. The question stands though."`
+			choice
+				`	"Oh sure, I'm willing to help."`
+					goto agree
+				`	"Sorry, but I don't think I want to get that involved."`
+					decline
+			label agree
+			`	"Fantastic," he says, suddenly grinning widely enough to be seen in your periphery as you walk along. "I'll add you to the list as soon as I get back to the office again. Your ID will also let you see which packages are ours specifically after that too. Oh and again, thank you!"`
```

Comment:
This spot in particular doesn't seem consistent with Wayde's characterization earlier in the conversation. He seems to be trying to hide his [eagerness, enthusiasm] and remain discreet until now. I can get him being a little more excited when the player agrees, but this feels a bit overboard - a huge grin feels almost like he's "breaking cover".

Reply:
Ah yes, now I see the problem with it. What do you think of this?
```suggestion
			`	"Fantastic," he says, quickly modulating his voice as someone briefly looks up at you from their phone. "I'll add you to the list as soon as I get back to the office again. Your ID will also let you see which packages are ours specifically after that too. Oh and again, thank you!"`
```

Reply:
That's an angle I wouldn't have gone for, but certainly not bad. Works for me!